ZenPacks/zenoss/Microsoft/Windows/datasources/EventLogDataSource.py

```diff
@@ -114,12 +114,23 @@ def config_key(cls, datasource, context):
     @classmethod
     def params(cls, datasource, context):
         te = lambda x: datasource.talesEval(x, context)
+        query = ""
+        query_error = True
+
+        if datasource.query.strip() != '$':
+            try:
+                query = te(' '.join(string_to_lines(datasource.query)))
+                query_error = False
+            except Exception:
+                pass
+
         return dict(
-            eventlog=te(datasource.eventlog), 
-            query=te(' '.join(string_to_lines(datasource.query))),
-            max_age=te(datasource.max_age), 
-            eventid=te(datasource.id)
-        )
+                eventlog=te(datasource.eventlog), 
+                query=query,
```

Comment:
If the above TALES evaluation raised an exception, `query` would be an empty string here. That wouldn't be very useful for understanding the event. It would be better to use the unevaluated datasource.query that the user configured if it wasn't able to be TALES evaluated.